### PR TITLE
Chart style updates

### PR DESCRIPTION
**Pull request by WPFerg**
Fixes 7/8 in #494. Indicator close buttons need a new SVG/png to implement, so I'm waiting on that.

Also fixes an issue in #497

### CODE REVIEW

File: src/showcase/showcase-changes.less [hunk 1]
```diff
@@ -179,11 +179,15 @@
     }
 }
 
+.multi.rsi.indicator {
+    stroke: @tertiary-purple;
+}
```
Comment (owennw) [inline src/showcase/showcase-changes.less]: I'm somewhat surprised not to see a companion change for this? Were we just always using the wrong colour?

**Comment by owennw** [comment]
When you say waiting on that, have you requested that change from anyone?

**Comment by WPFerg** [comment]
@owennw Rui

File: src/showcase/showcase-changes.less (hunk 1 above)
Comment (WPFerg) [inline src/showcase/showcase-changes.less]: I couldn't find any references to `@tertiary-purple`, and for some reason, I don't think Less is overriding it (or the selector's changed in Bitflux). I'll have a look

Comment (WPFerg) [inline src/showcase/showcase-changes.less]: Okay, for some reason it reset to D3FC's default, and `@tertiary-purple` was applied, but for some reason the order of precedence changed and the showcase CSS is being overriden by D3FC CSS.
Even though on Bitflux it works in the correct order of precedence.

Comment (owennw) [inline src/showcase/showcase-changes.less]: But the styles are correct after this PR?

Comment (WPFerg) [inline src/showcase/showcase-changes.less]: Yeah, I think it's just this instance where it happens

File: src/sidebars/favourites/minichart/minichart.less [hunk 2]
```diff
@@ -2,7 +2,7 @@
   position: absolute;
   padding-top: 10px;
   width: 100%;
-  overflow: hidden;
+  overflow: visible;
```
Comment (owennw) [inline src/sidebars/favourites/minichart/minichart.less]: What's the default for overflow?

Comment (WPFerg) [inline src/sidebars/favourites/minichart/minichart.less]: I believe `visible`, but in this instance it might be best to be explicit

**Pull request merged**